cdc/processor/sourcemanager/engine/pebble/event_sorter.go

```diff
@@ -113,10 +114,11 @@ func (s *EventSorter) RemoveTable(tableID model.TableID) {
 	s.mu.Lock()
 	if _, exists := s.tables[tableID]; !exists {
 		s.mu.Unlock()
-		log.Panic("remove an unexist table",
```

Comment:
When does it happen?

Reply:
Just align to `AddTable`. If it doesn't panic when add an exist table, it shouldn't panic when remove an unexist table, either.